### ISSUE TITLE
Review disclaimer text at beginning of the page

### ISSUE THREAD

**Issue by infokujur** [koroonakaart/koroonakaart#4]
Current:
Lehte arendadakse aktiivselt. Andmed pärinevad siit
Mis on koroonaviirus?
Lisafunktsionaalsuse ideed oodatud GitHub'i — https://github.com/okestonia/koroonakaart/issues

We should communicate - that it's really fresh project (so people are more forgiving), link somewhere were non techies can have conversation about it.

**Comment by teelepehk** [comment]
Pakun parema sõnastuse disclaimerile (ja praegu torkab disclaimer rohkem silma kui musta taustaga ülemine kirjeldusriba (mis võiks palju suuremas kirjas olla või siis disclaimeri sees kõik info): 

KOROONAVIIRUS EESTIS 
Statistika koroonaviiruse põhjustatud COVID-19 haiguse leviku kohta Eestis 
· Uuendatud: .... 
Mis on koroonaviirus?

Lehte arendatakse pidevalt. Andmed pärinevad [Terviseametist](https://www.terviseamet.ee/et/uuskoroonaviirus). 
Kui soovid lehte täiendada, pane ideed kirja GitHub'i — https://github.com/okestonia/koroonakaart/issues

**Comment by neuroactive** [comment]
For improved usability, it would be preferable to say "Andmed pärinevad Terviseametist" rather than "Andmed pärinevad siit" as then the user doesn't have to click to determine the source of the data.

**Comment by neuroactive** [comment]
The Github link would be better placed in a different part of the page information in my opinion. It won't be of interest to most members of the public. It increases the amount of text at the top of the page, meaning that the "Mis on koroonaviirus?" link is less prominent and the other page content is pushed further down.

I would put it somewhere at the bottom of the page for now, and in the future in an "About Koroonakaart" section, which should be added somewhere.

**Comment by neuroactive** [comment]
I also think we should remove the "Lehte arendadakse aktiivselt." line. Now that the site is functional, it simply serves no purpose and is reminiscent of the "Under construction" signs which were popular on websites in the 1990s (see the first answer here: https://ux.stackexchange.com/questions/26668/what-message-should-we-use-with-under-construction-page)

**Comment by neuroactive** [comment]
As part of making the site work better on mobiles, I have moved the summary text from the header to the disclaimer box, and have also removed the "Lehte arendadakse aktiivselt." line for now. Once we have an "About this site" section, we could mention that we aim to update the site at least on a daily basis. I have kept the date the site was last updated in the top-right corner of the page for now, as I think that is a reasonable place users might expect to see that information.
